aspnetcore/blazor/globalization-localization.md

```diff
@@ -52,6 +52,18 @@ The following field types have specific formatting requirements and aren't curre
 
 ## Localization
 
+### Blazor WebAssembly
+
+Blazor WebAssembly apps set the culture using the user's [language preference](https://developer.mozilla.org/docs/Web/API/NavigatorLanguage/languages).
+
+To explicitly configure the culture, set `CultureInfo.DefaultThreadCurrentCulture` and `CultureInfo.DefaultThreadCurrentUICulture` in `Program.Main`.
+
+By default, Blazor's linker configuration for Blazor WebAssembly apps strips out internationalization information except for locales explicitly requested. For more information and guidance on controlling the linker's behavior, see <xref:host-and-deploy/blazor/configure-linker#configure-the-linker-for-internationalization>.
+
+For a Blazor WebAssembly sample app with a culture picker, see the [LocSample](https://github.com/pranavkm/LocSample) localization sample app.
```

Comment:
Should I move this to the samples repo? Maybe we want to say something to the effect of, while the culture that Blazor picks by default might be sufficient for most users, consider offering a way for users to specify their preferred locale. <Sample link here> has an example of a doing this.

Reply:
For today ... given the workload at this time 🏃 ... it's fine to leave it where it is. If possible, I'd like to roll this into the common sample apps *soon-ish*. The work would be done after 3.2 releases and is tracked by https://github.com/dotnet/AspNetCore.Docs/issues/14216. The goal is to minimize the number of sample apps that we need to keep up with and make releases easier to manage. If you prefer it as a standalone sample, that's ok. Let me know. :ear: